packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1122,10 +1122,13 @@ class _TabBarViewState extends State<TabBarView> {
       _controller.animation.addListener(_handleTabControllerAnimationTick);
   }
 
+  List<Widget> ensureGlobalKeysForList(List<Widget> items) =>
```

Comment:
Also, according to our style guide `=>` is reserved for expressions that fit on one line. You'll have to use good old `{ }`here.